### ISSUE TITLE
TypeError: Cannot read property 'isFile' of undefined

### ISSUE THREAD

**Issue by clockworked247** [gregnb/filemanager-webpack-plugin#29]
Hi,

I'm seeing an error when attempting to use FileManagerPlugin.  This issue started appearing after updating to the latest version of the library.  

Changing /filemanager-webpack-plugin/lib/index.js:179
```
if (sStats.isFile()) {

to 

if (sStats && sStats.isFile()) {
```
Fixes the issue.

## Config

    new FileManagerPlugin({
      onStart: {
        delete: [
          config.build.copyToDirectory + 'static/vue/js/manifest.*',
          config.build.copyToDirectory + 'static/vue/js/app.*',
          config.build.copyToDirectory + 'static/vue/js/vendor.*',
          config.build.copyToDirectory + 'static/vue/css/app.*',
        ]
      },
      onEnd: {
        copy: [
          { source: './dist/project/static/vue/js/.', destination: config.build.copyToDirectory + 'static/vue/js' },
          { source: './dist/project/static/vue/css/.', destination: config.build.copyToDirectory + 'static/vue/css' },
          { source: './dist/index.html', destination: config.build.copyToDirectory + 'index.html' }
        ]
      }
    })

## Issue
/node_modules/filemanager-webpack-plugin/lib/index.js:179
                        if (sStats.isFile()) {
                                  ^

TypeError: Cannot read property 'isFile' of undefined
    at /Users/sbrooks/Desktop/learning/vue-qconsole/node_modules/filemanager-webpack-plugin/lib/index.js:179:35
    at FSReqWrap.oncomplete (fs.js:114:15)

## Your Environment

| Tech         | Version |
|--------------|---------|
| filemanager-plugin-webpack  |   1.0.26      |
| node        |    v7.9.0     |
| OS      |   Mac      |

**Comment by gregnb** [comment]
@clockworked247 Would you like to send a PR in for this issue?

**Comment by gregnb** [comment]
I'll make an update but this issue is highlighting something else. You are trying to copy something that does not exist. Remove the dot at the end and change your config to:

```
  onEnd: {
    copy: [
      { source: './dist/project/static/vue/js/', destination: config.build.copyToDirectory + 'static/vue/js' },
      { source: './dist/project/static/vue/css/', destination: config.build.copyToDirectory + 'static/vue/css' },
      { source: './dist/index.html', destination: config.build.copyToDirectory + 'index.html' }
    ]
  }
```

I'll add better error handling

**Comment by clockworked247** [comment]
@gregnb Ok, will do.  Thanks.

**Comment by eamodio** [comment]
@gregnb I'm running into this same issue when trying to copy a file that may or may not exist. I only want to copy it if it exists, and if it doesn't I don't care. What is the recommended way to get that to work?

**Comment by gregnb** [comment]
Just published a fix for this in filemanager-webpack-plugin@1.0.29

**Comment by xiaoyouyu** [comment]
the bug is i has not ./dist file，cao
